fixed-point-test.rkt

```diff
@@ -0,0 +1,12 @@
+#lang racket
+
+(require rackunit
+         "fixed-point.rkt")
+
+(define (inc x) (+ x 1))
```

Comment:
just a tip: Scheme already defines the function _add1_ and _sub1_ in the standard.

Reply:
this one was on purpose...I wanted to implement as much as I could by hand...just for fun =D